src/pages/billing/Project.js

```diff
@@ -336,9 +336,11 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
             key: member.email,
             adminLabel: billingRoles.owner,
             userLabel: billingRoles.user,
-            member, adminCanEdit,
+            member,
+            adminCanEdit: (projectHasMultipleOwners && isOwner),
             onEdit: () => setEditingUser(member),
-            onDelete: () => setDeletingUser(member)
+            onDelete: () => setDeletingUser(member),
+            isOwner
           })
         }, _.orderBy([sort.field], [sort.direction], projectUsers))
         )
```

Comment:
@cahrens are there any concerns or considerations that I need to be aware of for the spend reporting tab? aside from the feature flag, is that also limited only to people with the owner role on the project? if so, I can add that logic into this PR

Reply:
At this point, we need to limit the spend report visibility to owners. Otherwise, they will just get an error. We know that it would be good to open the report to non-owners in the future, but that will require us to change the query.

![image](https://user-images.githubusercontent.com/484484/161979691-a8c6e424-1097-4556-b15f-a5d0d82d297a.png)

Reply:
Please extend the [spend report test](https://github.com/DataBiosphere/terra-ui/blob/dev/integration-tests/tests/billing-projects.js) to verify that the spend report does not show for non-owners, even if they are in the alpha spend group.

Reply:
Functionality resolved in 45e4330b, test extension to come soon